src/commands/app/run.js

```diff
@@ -48,7 +65,14 @@ class Run extends BaseCommand {
     process.env.REMOTE_ACTIONS = !flags.local
     const scripts = AppScripts({ listeners })
     try {
-      await scripts.runDev()
+      const options = {
+        https: {
+          cert: PUB_CERT_PATH, // Path to custom certificate
+          key: PRIVATE_KEY_PATH // Path to custom key
+        },
+        logLevel: flags.verbose ? 4 : 3
```

Comment:
hey @purplecabbage just wondering is there some use case to log other things that errors here ? (loglevel = 1)

Reply:
I like to see what is being served myself ... I can change it to 4 || 1

Reply:
ok fair point 🙂